src/coreComponents/python/modules/geosx_xml_tools_package/geosx_xml_tools/tests/test_manager.py

```diff
@@ -235,40 +138,20 @@ class TestXMLProcessor(unittest.TestCase):
     def setUpClass(cls):
         generate_test_xml.generate_test_xml_files('.')
 
-    def diff_xml(self, source, target):
-        self.assertTrue(filecmp.cmp(source, target))
-
-    def test_no_advanced_xml(self):
-        tmp = xml_processor.process('./no_advanced_features_input.xml',
-                                    outputFile='./no_advanced_features_processed.xml',
-                                    verbose=0,
-                                    keep_parameters=False,
-                                    keep_includes=False)
-        self.diff_xml(tmp, './no_advanced_features_target.xml')
-
-    def test_parameters_xml(self):
-        tmp = xml_processor.process('./parameters_input.xml',
-                                    outputFile='./parameters_processed.xml',
-                                    verbose=0,
-                                    keep_parameters=False,
-                                    keep_includes=False)
-        self.diff_xml(tmp, './parameters_target.xml')
-
-    def test_includes_xml(self):
-        tmp = xml_processor.process('./included_input.xml',
-                                    outputFile='./included_processed.xml',
-                                    verbose=0,
-                                    keep_parameters=False,
-                                    keep_includes=False)
-        self.diff_xml(tmp, './included_target.xml')
-
-    def test_symbolic_xml(self):
-        tmp = xml_processor.process('./symbolic_parameters_input.xml',
-                                    outputFile='./symbolic_parameters_processed.xml',
-                                    verbose=0,
-                                    keep_parameters=False,
-                                    keep_includes=False)
-        self.diff_xml(tmp, './symbolic_parameters_target.xml')
+    @parameterized.expand([['no_advanced_features', './no_advanced_features_input.xml', './no_advanced_features_target.xml'],
+                           ['parameters', './parameters_input.xml', './parameters_target.xml'],
+                           ['included', './included_input.xml', './included_target.xml'],
+                           ['symbolic', './symbolic_parameters_input.xml', './symbolic_parameters_target.xml']])
+    def test_sequence(self, name, input_file, target_file, expect_fail=False):
```

Comment:
Here too, `name` is not a parameter of the test.
Would relying on the `xml` file name be enough?
Or simply a comment at the end of the line?